src/configuration/ConfigEditor.tsx

```diff
@@ -131,7 +134,7 @@ export function overhaulStyles(theme: GrafanaTheme2) {
       font-weight: 400;
     `,
     container: css`
-      maxWidth: 578;
+      maxwidth: 578;
```

Comment:
doesn't look like `container` is used anywhere. can it be removed? if not it should be `max-width` instead.